pkg/cloud/aws/aws.go

```diff
@@ -76,3 +77,32 @@ func NewProviderAssets(config config.OperatorConfig) (common.CloudProviderAssets
 	}
 	return assets, nil
 }
+
+// CloudConfigTransformer implements the cloudConfigTransformer. It uses the source ConfigMap and
+// infra.status.platformStatus.AWS.ServiceEndpoints to create a new config that include the
+// ServiceOverrides sections. The transformer uses infra.status.platformStatus.region as the
+// signing service for all the ServiceOverrides.
+// It returns an error if the platform is not AWSPlatformType.
+func CloudConfigTransformer(source string, infra *configv1.Infrastructure) (string, error) {
+	if !(
+		infra.Status.PlatformStatus != nil &&
+		infra.Status.PlatformStatus.Type == configv1.AWSPlatformType) {
```

Comment:
Nit, I'm not sure about this indentation, it looks weird to me, hard to find the end of the `(`

Is think this might be clearer?
```suggestion
	if !(infra.Status.PlatformStatus != nil &&
		infra.Status.PlatformStatus.Type == configv1.AWSPlatformType) {
```

Reply:
Sure, done